app/assets/javascripts/remote_consoles/webmks.js

```diff
@@ -0,0 +1,55 @@
+//= require jquery
+//= require jquery-ui/ui/unique-id.js
+//= require jquery-ui/ui/widget.js
+//= require jquery-ui/ui/widgets/dialog.js
+//= require jquery-ui/ui/widgets/button.js
+//= require_tree ../locale
+//= require gettext/all
+
+$(function() {
+  // WebMKS cannot be a part of the asset pipeline, therefore, it has to be loaded with this hack
+  var link = document.createElement('link');
```

Comment:
May be missing something, but why wouldn't we just use jQuery to perform modifications on the DOM, since we have it imported?

Reply:
Well, we load jquery because the webmks console needs it, I don't really want to use it when not necessary.